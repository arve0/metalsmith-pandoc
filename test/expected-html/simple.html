<p>This is a markdown dummy file for testing</p>
<h1 id="header-1">Header 1</h1>
<p>text <em>italics</em> <strong>bold</strong></p>
<div class="sourceCode"><pre class="sourceCode python"><code class="sourceCode python"><span class="cf">for</span> f <span class="op">in</span> files:
    <span class="bu">print</span>(f)</code></pre></div>
<h2 id="header2" class="withclass">Header2</h2>
