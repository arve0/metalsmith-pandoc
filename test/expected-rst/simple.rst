This is a markdown dummy file for testing

Header 1
========

text *italics* **bold**

.. code:: python

    for f in files:
        print(f)

Header2
-------

