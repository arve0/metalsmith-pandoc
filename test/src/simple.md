This is a markdown dummy file for testing

# Header 1
text
*italics*
**bold**

```python
for f in files:
    print(f)
```

## Header2 {.withclass}
