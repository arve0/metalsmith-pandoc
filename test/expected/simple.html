<p>This is a markdown dummy file for testing</p>
<h1 id="header-1">Header 1</h1>
<p>text <em>italics</em> <strong>bold</strong></p>
<pre class="sourceCode python"><code class="sourceCode python"><span class="kw">for</span> f in files:
    <span class="dt">print</span>(f)</code></pre>
<h2 id="header2" class="withclass">Header2</h2>
